examples/python/l2-resources/datadog_integration.py

```diff
@@ -0,0 +1,25 @@
+from aws_cdk import (
+  # Duration,
+  Stack,
+  # aws_sqs as sqs,
+)
+from constructs import Construct
+
+from awscdk_resources_mongodbatlas import ( DatadogIntegration, DatadogRegion )
+
+class CdkTestAppPython2Stack(Stack):
+
+  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+    super().__init__(scope, construct_id, **kwargs)
+
+    projectt_id = self.node.try_get_context('project_name')
+    apii_key = self.node.try_get_context('org_id')
```

Comment:
[q] why do we try to get org_id and name it apii_key in l2? (might be lacking context)

Reply:
typo error when copying from another IDE:( fixed